kamon-core/src/main/scala/kamon/trace/Span.scala

```diff
@@ -174,14 +197,14 @@ object Span {
   val BooleanTagTrueValue = "1"
```

Comment:
This was related to tag values, which might be either numbers, booleans or strings. The new implementation takes care of it.